client/Dockerfile.webpack

```diff
@@ -0,0 +1,24 @@
+ARG image_tag=latest
+
+FROM liberoadmin/reviewer_client_packages:${image_tag}
+
+WORKDIR /opt
+
+COPY globals.d.ts .
```

Comment:
can be a single `COPY` instead of 6 with a target of `./`, creating a single layer?